# Hekateros hekateros_control/IsAlarmed service

---
bool is_alarmed   # robot is [not] alarmed
